packages/ui/Layout/Cell.js

```diff
@@ -59,6 +59,17 @@ const CellBase = kind({
 		 */
 		component:  EnactPropTypes.renderable,
 
+		/**
+		 * Customizes the component used in the `Cell` by mapping the supplied collection of CSS class names to the corresponding internal elements and states of this component.
+		 *
+		 * When using {@link ui/Layout.Cell}, the `css` prop is forwarded to this component
```

Comment:
"this component" makes a big misunderstanding. It can be read like "if a developer set `componentCss` prop to this component, then `css` prop will be forwarded to this component."